web/frps/Makefile

```diff
@@ -4,4 +4,4 @@ build:
 	@npm run build
```

Comment:
We should provide build scripts here to let others build it in a simple way.

I run `make build` on my machine and it failed. If it relies on other tools, we'd better descirbe it on README.

Reply:
I just edited the `Makefile` to  `install` before `build`

Reply:
`yarn install` is used in README.md, what's the difference?

Reply:
It was automatically generated and has now been edited